components/layout_2020/style_ext.rs

```diff
@@ -494,3 +494,30 @@ fn size_to_length(size: &Size) -> LengthPercentageOrAuto {
         Size::Auto => LengthPercentageOrAuto::Auto,
     }
 }
+
+pub(crate) fn transferred_inline_min_max(
+    ratio: f32,
+    style: &ComputedValues,
+    containing_block: &ContainingBlock,
+    pbm: &PaddingBorderMargin,
+) -> (Length, Option<Length>) {
+    let max_block_size = style
+        .max_box_size(containing_block.style.writing_mode)
+        .block
+        .map(|l| l.maybe_percentage_relative_to(containing_block.block_size.non_auto()))
+        .flatten()
+        .map(|l| l);
+    let min_block_size = style
+        .min_box_size(containing_block.style.writing_mode)
+        .block
+        .maybe_percentage_relative_to(containing_block.block_size.non_auto())
+        .auto_is(|| Length::zero());
+
+    match style.get_position().box_sizing {
+        BoxSizing::ContentBox => (min_block_size / ratio, max_block_size.map(|b| b / ratio)),
+        BoxSizing::BorderBox => (
+            min_block_size / ratio - pbm.padding_border_sums.inline,
```

Comment:
Is `ratio` for the inner sizes, the outer sizes, or depends on `box-sizing`?

 - If it's for the inner sizes, this case seems wrong since `min_block_size` is an outer size.
 - If it's for the outer sizes, it's the same problem for the `content-box` case
 - If it depends on `box-sizing`, then `transferred_size_suggestion` needs to check the `box-sizing`